src/test/java/net/snowflake/client/TestUtil.java

```diff
@@ -66,13 +66,14 @@ public interface TestRunInterface {
    * System.getenv wrapper. If System.getenv raises an SecurityException, it is ignored and returns
    * null.
    *
-   * <p>This is replicated from SnowflakeUtil.systemGetEnv, because the old driver doesn't have that
-   * function for the tests to use it. Replace this function call with SnowflakeUtil.systemGetEnv
-   * when it is available.
-   *
+   * @deprecated This method should be replaced by SnowflakeUtil.systemGetEnv.
+   *     <p>This is replicated from SnowflakeUtil.systemGetEnv, because the old driver doesn't have
+   *     that function for the tests to use it. Replace this function call with
+   *     SnowflakeUtil.systemGetEnv when it is available.
    * @param env the environment variable name.
    * @return the environment variable value if set, otherwise null.
    */
+  @Deprecated
```

Comment:
This is a test method, you can remove it without deprecation as it is used only within the tests.

Reply:
@sfc-gh-pbulawa The idea is marked as deprecated for now because is required to execute the test with the old driver, but at some moment we will replace with the SnowflakeUtil.systemGetEnv.